src/components/ViewDeck/ViewDeck.js

```diff
@@ -97,228 +80,205 @@ class ViewDeck extends Component {
-          <div>
-            <img
-              className="featuredViewDeckCard"
-              src={this.state.hoverCard}
-              width="200px"
-              height="280"
-            />
-          </div>
-
-          <br />
-          <h4 className="editDeckHeader">Total Cards: {this.state.deckQty}</h4>
+  const includedCards = props.reduxStore.cardList.filter(
+    (card) => card.deckid === props.reduxStore.selectedDeck.id
+  );
+  // console.log("includedCards:", includedCards);
+  const featuredCard = includedCards.filter(
+    (featured) => featured.is_featured === true
+  );
+  const parsedFeaturedCard = featuredCard.map((fCard) =>
+    JSON.parse(fCard.api_data)
+  );
+  // console.log("parsedFeaturedCard", parsedFeaturedCard);
+  const featuredUri = parsedFeaturedCard.map((fUri) => fUri.image_uris.normal);
+
+  countQty(includedCards);
+
+  const parsedCards = parseCards(includedCards);
+
+  const devotion = devotionCalc(parsedCards);
+
+  const cmc = cmcCalc(parsedCards);
+
+  return (
+    <div>
+      <h1 className="editDeckHeader">
+        Viewing {props.reduxStore.selectedDeck.deckname} from{" "}
+        {props.reduxStore.user.username}{" "}
+      </h1>
+
+      <div className="viewDeckEditDeckView">
+        <div>
+          <img
+            className="featuredViewDeckCard"
+            src={hoverCard}
+            width="200px"
+            height="280"
+          />
+        </div>
 
-          <table>
-            <thead>
-              <tr>
-                <th></th>
-                {/* Hovercard^ */}
-                <th>Quantity</th>
-                <th className="nameTh">Card Name</th>
-                {/* <th>isCMDR?</th> */}
-                <th></th>
-                {/* isFeatured? */}
-                <th></th>
-                {/* Type */}
-                {/* <th>DELETE</th> */}
+        <br />
+        <h4 className="editDeckHeader">Total Cards: {state.deckQty}</h4>
+
+        <table>
+          <thead>
+            <tr>
+              <th></th>
+              {/* Hovercard^ */}
+              <th>Quantity</th>
+              <th className="nameTh">Card Name</th>
+              {/* <th>isCMDR?</th> */}
+              <th></th>
+              {/* isFeatured? */}
+              <th></th>
+              {/* Type */}
+              {/* <th>DELETE</th> */}
+            </tr>
+          </thead>
+          <tbody>
+            {includedCards.map((card) => (
+              <tr key={card.id}>
+                <td></td>
+                {/* //hovercard^ */}
+                <td className="qtyTd">x {card.quantity}</td>
+                <td
+                  className="nameTd"
+                  onMouseOver={() => cardDisplay(card)}
+                  onMouseLeave={() => removeDisplay(card)}
+                >
+                  {card.name}
+                </td>
+
+                {/* <td>{card.is_cmdr}</td> */}
+                <td>{card.is_featured}</td>
+                <td></td>
```

Comment:
empty cells? I get that they seem to match the empty header columns, but why? Also why the commented out sections?